cluster/config-defaults.yaml

```diff
@@ -98,3 +98,11 @@ teapot_admission_controller_ignore_namespaces: "^kube-system$"
 {{if ne .Environment "production"}}
 etcd_instance_count: "3"
 {{end}}
+
+# vertical pod autoscaler
+{{if ne .Environment "dev"}}
```

Comment:
We only have `test` and `production`. Besides, I'd enable it explicitly in a couple of dev clusters first, rather than automatically enabling it in all test clusters. Or do you think that it's fine?

Reply:
Oh I think we should disable it by default then.